src/main/java/com/treasure/hunt/strategy/geom/GeometryItem.java

```diff
@@ -26,6 +25,9 @@
     @Getter
     private List<GeometryStyle> geometryStyles = new ArrayList<>();
     private int preferredStyle = 0;
+    @Getter
+    @Setter
+    private boolean isSelected = false;
```

Comment:
youre right, got a better solution now.

Reply:
Ill fix this on branch [bugfix/#183-missing-linesegment-in-linestring](https://github.com/jotoh98/treasureHunt/tree/bugfix/%23183-missing-linesegment-in-linestring), since fixing this is almost impossible, as long as issue #183 ist not fixed.